src/actions/newThought.ts

```diff
@@ -241,7 +242,9 @@ export const newThoughtActionCreator =
     // cancel if tutorial has just started
     if (tutorial && tutorialStep === TUTORIAL_STEP_START) return
 
-    if (!preventSetCursor && isTouch) {
+    // asyncFocus can cause disruptive autoscroll behavior, so it's important to avoid it when the selection is already on a thought.
```

Comment:
Why does `asyncFocus` disrupt autoscroll behavior? I wonder if it would help if the `hiddenInput` was positioned in the center of the viewport.

It would be nice to have a more resilient `asyncFocus` that worked regardless of the selection state before it is called, but maybe that's too ambitious.